src/Register.hs

```diff
@@ -11,3 +12,21 @@ type Register = FixedVector 32 Bit
 -- create a Register from a list of Bits, padding missing elements with 0 and ignoring surplus elements
 fromList :: [Bit] -> Register
 fromList = fromListWithDefault O
+
+not :: Register -> Register
+not = fmap Bit.not
+
+and :: Register -> Register -> Register
+and x y = fromList (fmap (uncurry Bit.and) (zip (toList x) (toList y)))
```

Comment:
@kirschmet2 when giving `fmap` both parameters, consider using its operator form instead. This way you can even omit parentheses on both parameters.

Reply:
Thanks for the suggestion, I did so and added the functions for the other operations from Bit as well in a separate PR.